assets/charts/control-plane/calico/templates/felixconfig.yaml

```diff
@@ -0,0 +1,11 @@
+# General
+# =======
+---
+# Disable default failsafe inbound rules
```

Comment:
I think those lines can be removed all together, as they do not make sense without the context of Calico host protection file.